steps/synopsys-pt-power/configure.yml

```diff
@@ -0,0 +1,65 @@
+#=========================================================================
+# Synopsys PrimePower -- Gate-Level Power Estimation
+#=========================================================================
+# This step runs gate-level average power analysis with Synopsys PrimeTime PX
+# or PrimePower, the successor of the PrimeTime PX feature
+#
+# Both average and time-base power analysis can be done
+# Either gate-level or rtl switching activities can be used
+#
+# Author : Maximilian Koschay
+# Date   : 05.03.2021
+#
+
+name: synopsys-pt-power
+
+#-------------------------------------------------------------------------
+# Inputs and Outputs
+#-------------------------------------------------------------------------
+
+inputs:
+  - adk
+  - design.vcs.v
+  - design.pt.sdc
+  - design.spef.gz
+  - run.saif
+  - run.vcd
+  - design.namemap
+
+outputs:
+  - primetime.session
+  - design.sdf
+
+#-------------------------------------------------------------------------
+# Commands
+#-------------------------------------------------------------------------
+
+commands:
+  - ./run.sh
+
+debug:
+  - ./debug.sh
+
+#-------------------------------------------------------------------------
+# Parameters
+#-------------------------------------------------------------------------
+
+parameters:
+  design_name: undefined
+  #The strip path/ saif_instance must be defined, without any quotations!
+  saif_instance: th/dut
+  # analysis mode either averaged or time_based
+  analysis_mode: averaged
+  # If VCD activity files is the result of a zero-delay simulation, it needs
+  # to be specified! (Otherwise SDF simulation will be assumed)
+  zero_delay_simulation: False
+  # Set library operating condition
+  lib_op_condition: undefined
```

Comment:
Both parameters are used.
The zero_delay_simulation parameter is relevant to time_based analysis, where the program requires to know if the switching annotation is the result of a zero-delay or a delay annotated simulation. 
```
if {$ptpx_zero_delay_simulation == True} {
			echo "Read VCD activity annotation file from zero-delay simulation."
			read_vcd -strip_path $ptpx_strip_path -zero_delay $ptpx_vcd 
		} else {
			echo "Read VCD activity annotation file from SDF annotated simulation."
			read_vcd -strip_path $ptpx_strip_path $ptpx_vcd 
		}
```

The lib_op_condition defines which operation condition of the target/link libraries is used. Since the designer interface selects all .db in the ADK, best case and worst case libs are also included when available. If you do not specify a op condition the tool will choose one. 
```
if {$ptpx_op_condition != "undefined"} {
	set_operating_condition $ptpx_op_condition
} else {
	echo "No operating condition is defined. PrimeTime will choose one."
}
```

Reply:
Right. However, just from glancing at the configure.yml (which is how many users will learn what a node does), it isn't clear how to use the lib_op_condition parameter. Maybe we can make it a bit clearer with a comment that says the conditions are pulled from the dbs by default, but you can override the choice from here.

Reply:
Yes, I agree. I have changed it in the last commit